src/client/Microsoft.Identity.Client.Broker/WamAdapters.cs

```diff
@@ -182,6 +182,26 @@ private static void AddPopParams(AuthenticationRequestParameters authenticationR
             }
         }
 
+        public static MsalTokenResponse AcquireTokenFromBroker(NativeInterop.AuthResult authResult,
```

Comment:
Rename to something like HandleResponse, ProcessResponse, ParseResponse.
nit: put first param on new line.